src/iedit.cpp

```diff
@@ -2370,6 +2414,26 @@ void iedit_parse(struct descriptor_data * d, const char *arg)
             case CYB_REFLEXTRIGGER:
             case CYB_SKULL:
             case CYB_TORSO:
+            case CYB_STRENGTHMOD:
+            case CYB_QUICKNESSMOD:
+            case CYB_ARMORMOD:
+            case CYB_CYBERARMGYROMOUNT:
```

Comment:
Does this make sense? This implies that there is cyberware of the type 'strengthmod' that can be installed, and that it should use the cyberarms table (same for qui, armo, gyro). Likely need to remove these.

Reply:
Yeh, think this was an earlier attempt I forgot to remove (I got a little drunk halfway through, as the title of this push implies) before I realized they were gonna be added as mods instead of standalone cyberware pieces. Went ahead and removed em.